src/main/java/bral/museum/RijksSearchFrame.java

```diff
@@ -0,0 +1,88 @@
+package bral.museum;
+
+import javax.swing.*;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import java.awt.*;
+
+public class RijksSearchFrame extends JFrame
+{
+    private JTextField searchField;
+    private JButton nextButton;
+    private JButton prevButton;
+    private int currentPage = 1;
+    private String currentQuery = "";
```

Comment:
This doesn't need to be a member variable.

Reply:
Thank you for the feedback. I just want to clarify, can I leave them as fields so that I can access them in other methods?

Reply:
Yes.